compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeBuilder.java

```diff
@@ -649,11 +651,11 @@ public BLangNode transform(ModulePartNode modulePart) {
         this.currentCompilationUnit = compilationUnit;
         compilationUnit.name = currentCompUnitName;
         compilationUnit.setPackageID(packageID);
-        Location pos = getPosition(modulePart);
+        modulePartPos = getPosition(modulePart);
```

Comment:
Can't we avoid maintaining this at class level and instead set it for xmlns decl statements here similar to imports?

Can be an issue with xmlns-decl-stmt (not at module-level), but for those, do we need compUnit?